lib/command.c

```diff
@@ -471,87 +471,95 @@ static int config_write_host(struct vty *vty)
 	if (host.name)
 		vty_out(vty, "hostname %s\n", host.name);
 
-	if (host.encrypt) {
-		if (host.password_encrypt)
-			vty_out(vty, "password 8 %s\n", host.password_encrypt);
-		if (host.enable_encrypt)
-			vty_out(vty, "enable password 8 %s\n",
-				host.enable_encrypt);
-	} else {
-		if (host.password)
-			vty_out(vty, "password %s\n", host.password);
-		if (host.enable)
-			vty_out(vty, "enable password %s\n", host.enable);
-	}
+	/* The following are all configuration commands that are not sent to
+         * watchfrr.  For instance watchfrr is hardcoded to log to syslog so
+         * we would always display 'log syslog informational' in the config
+         * which would cause other daemons to then switch to syslog when they
+         * parse frr.conf.
+         */
+	if (strcmp(zlog_default->protoname, "WATCHFRR")) {
```

Comment:
This is pretty ugly... (and the comment is incorrect, the commands *are* sent to watchfrr)

I would say it's way better to just not collect any config from watchfrr in vtysh's `vtysh_write_config_integrated()`, considering that we're not loading it into watchfrr either at this point.